#request fields
string req
---
#response fields
string res
